### PR TITLE
Fix exit code when dotnet test fails

### PR DESCRIPTION
**Pull request by AndcultureCode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +114,7 @@ best performance.
 * `and-cli dotnet` - Runs the dotnet solution's web project
     * `and-cli dotnet -b, --build` - Builds the solution
     * `and-cli dotnet -c, --clean` - Cleans the solution
+    * `and-cli dotnet -- --cli my command` - Runs commands through a project's custom dotnet cli project
```
Comment (dylanjustice) [inline README.md]: Is there an extra `--` in this description?

File: README.md [hunk 2]
```diff
@@ -136,4 +151,17 @@ While the dotnet core cli provides some nuget commands, the process start to fin
...
+
+## Troubleshooting
+
+### Value for command arguments are out of order
+Depending upon the shell/terminal you are using, the node process sometimes requires the `--` delimiter between the command and the arguments. Otherwise, especially in a shell like windows command prompt, the value for arguments gets piped out of order.
+
+Example:
+* Before: `and-cli dotnet --cli "test db migrate"`
+    * Works in most shells, but requires the arguments to be in quotes. Fails in windows command prompt
+* After: `and-cli dotnet -- --cli test db migrate`
```
Comment (dylanjustice) [inline README.md]: I think this answers my above question

File: cli-dotnet-test.js [hunk 3]
```diff
@@ -56,12 +56,16 @@ const dotnetTest = {
...
 
-        shell.echo(formatters.dotnet(stdout));
-        shell.echo(formatters.dotnet(stderr));
+        shell.echo(formatters.dotnet(result.stdout));
+        shell.echo(formatters.dotnet(result.stderr));
 
         dir.popd();
+
+        if (result.code !== 0) {
+            shell.exit(1);
```
Comment (myty) [inline cli-dotnet-test.js]: Not sure if it really matters much, but If you do `shell.exit(result.code);`, you will return the same error code from command that caused it.

Comment (wintondeshong) [reply cli-dotnet-test.js]: Damn good point

File: README.md (hunk 2 above)
Comment (wintondeshong) [reply README.md]: Yeah, this is intentional and applies to all commands. Just so happens this is the first to have the multiple values to an option

**Pull request merged**